tests/test_runes.py

```diff
@@ -594,6 +594,17 @@ def test_missing_method_or_nodeid(node_factory):
         l1.rpc.checkrune(rune=rune4, nodeid=l1.info['id'])
 
 
+def test_rune_method_not_equal_and_method_empty(node_factory):
+    """Test `checkrune` when `method` parameter is the empty string and a `method` is negated in the rune."""
+    l1 = node_factory.get_node()
+    rune = l1.rpc.createrune(restrictions=[["method/getinfo"]])['rune']
+
+    with pytest.raises(RpcError, match='Not permitted: method is equal to getinfo'):
+        l1.rpc.checkrune(rune=rune, method='getinfo')
+    assert l1.rpc.checkrune(rune=rune, method='invoice')['valid'] is True
+    with pytest.raises(RpcError, match='Not permitted: method not present'):
+        l1.rpc.checkrune(rune=rune, method='')
+
 def test_invalid_restrictions(node_factory):
```

Comment:
give another endline to fix the CI 

```
2023-10-10T18:07:28.4276716Z checking for egrep... /usr/bin/grep -E
2023-10-10T18:07:28.9992884Z checking for ANSI C header files... yes
2023-10-10T18:07:29.1313563Z checking for sys/types.h... yes
2023-10-10T18:07:29.3154822Z checking for sys/stat.h... yes
2023-10-10T18:07:29.4359691Z checking for stdlib.h... tests/test_runes.py:608:1: E302 expected 2 blank lines, found 1
2023-10-10T18:07:29.5113573Z yes
2023-10-10T18:07:29.5341322Z make: *** [Makefile:538: check-python-flake8] Error 1
2023-10-10T18:07:29.5342110Z make: *** Waiting for unfinished jobs....
2023-10-10T18:07:29.7553430Z checking for string.h... yes
```

Reply:
Done!

I forgot to run `make check-python-flake8` before commiting :(

Reply:
Tested with f0f9bb23307c7894995cdb89c3ac762c3bf3796e.